content/250-postgres/1100-integrations/500-vscode-agent.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: 'VS Code Agent' 
+description: 'Streamline Prisma ORM workflows, manage migrations, and create databases directly in Visual Studio Code using the Prisma Agent.' 
+metaTitle: 'Visual Studio Code Agent' 
+metaDescription: 'Manage Prisma ORM migrations and provision Prisma databases efficiently within Visual Studio Code using the Prisma Agent.' 
+tocDepth: 3 
+toc: true
+---
+
+## Overview
+
+[Visual Studio Code](https://code.visualstudio.com/docs/copilot/chat/chat-agent-mode) includes an **agent** mode (powered by GitHub Copilot) that automatically performs code changes and executes Prisma CLI commands based on your prompts.
+
+## Capabilities
+
+The [Prisma Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=Prisma.prisma) enables the agent to perform the following tasks using our Model Context Protocol (MCP) server:
+
+- Manage Prisma Data Platform projects.
+- Provision new databases.
+- Run migrations and generate the Prisma Client.
+- Generate Prisma schemas and client code.
+- Execute Prisma commands directly in the terminal.
+
+This integration removes the need to manually run MCP commands, significantly speeding your database-related workflows.
```

Comment:
Also wouldn't say this imo. It's more like "Instead of running and configuring the Prisma MCP server explicitly, the tools are now backed into the VSCode Extension itself."